### ISSUE TITLE
Fate, Up Against Your Will

### ISSUE THREAD

**Issue by ManiaLightmare** [WoWManiaUK/Redemption#1553]
Same issue as this closed item.
https://github.com/WoWManiaUK/Redemption/issues/143

Jaina is not in Stormwind to give portal to Ogrimaar.

Quest worked a few days prior for another character of mine.  GM was not able to resolve by changing phase either.  Cleared cache as well.  Still could never see her.  Even after having the GM flag the quest as completed and the followup, she was still not in SW for me.

**Comment by captainloozer** [comment]
Hi ManiaLightmare and thank you for your report.

Unfortunately, I was not able to reproduce this on the dev server.  I recently tested another issue to a quest following this one.  Jaina was not showing up for me either until I completed ALL of the quests prior to this one.  Below is a video showing that I was able to do it, but I would suggest going back and double checking to make sure you have done all the quests in the chain leading up to this one.

https://youtu.be/fefEbQVG2K4

http://www.wowhead.com/quest=13347/reborn-from-the-ashes

Check out the top comment here and make sure you have done all the quests in the chain prior to you taking the shield back to Varian.
